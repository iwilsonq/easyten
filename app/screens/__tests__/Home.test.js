import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import HomeScreen from '../Home';

describe('Home Screen', () => {
	const wrapper = renderer.create(<HomeScreen />).toJSON();
	it('renders', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
