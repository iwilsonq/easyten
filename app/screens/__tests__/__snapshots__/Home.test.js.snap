// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen renders 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      centered={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#dededf",
            "fontFamily": "montserrat-semibold",
            "fontSize": 18,
            "fontWeight": "600",
            "paddingBottom": 8,
          },
          Object {
            "textAlign": "center",
          },
        ]
      }
    >
      Today
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      centered={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#dededf",
            "fontFamily": "montserrat-regular",
            "fontSize": 14,
          },
          Object {
            "textAlign": "center",
          },
        ]
      }
    >
      Sunday, January 20, 2018
    </Text>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flex": 1,
            "width": "100%",
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#9e9ea0",
              "height": 1,
              "marginVertical": 20,
              "width": 120,
            },
          ]
        }
      />
    </View>
  </View>
  <View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#32343a",
              "borderRadius": 4,
              "margin": 10,
              "padding": 16,
            },
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#dededf",
                "fontFamily": "montserrat-semibold",
                "fontSize": 18,
                "fontWeight": "600",
                "paddingBottom": 8,
              },
            ]
          }
        >
          1:21:13
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#dededf",
                "fontFamily": "montserrat-regular",
                "fontSize": 14,
              },
            ]
          }
        >
          Active Time
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#32343a",
              "borderRadius": 4,
              "margin": 10,
              "padding": 16,
            },
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#dededf",
                "fontFamily": "montserrat-semibold",
                "fontSize": 18,
                "fontWeight": "600",
                "paddingBottom": 8,
              },
            ]
          }
        >
          1:21:13
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#dededf",
                "fontFamily": "montserrat-regular",
                "fontSize": 14,
              },
            ]
          }
        >
          Active Time
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#32343a",
              "borderRadius": 4,
              "margin": 10,
              "padding": 16,
            },
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#dededf",
                "fontFamily": "montserrat-semibold",
                "fontSize": 18,
                "fontWeight": "600",
                "paddingBottom": 8,
              },
            ]
          }
        >
          1:21:13
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#dededf",
                "fontFamily": "montserrat-regular",
                "fontSize": 14,
              },
            ]
          }
        >
          Active Time
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#32343a",
              "borderRadius": 4,
              "margin": 10,
              "padding": 16,
            },
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#dededf",
                "fontFamily": "montserrat-semibold",
                "fontSize": 18,
                "fontWeight": "600",
                "paddingBottom": 8,
              },
            ]
          }
        >
          1:21:13
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#dededf",
                "fontFamily": "montserrat-regular",
                "fontSize": 14,
              },
            ]
          }
        >
          Active Time
        </Text>
      </View>
    </View>
  </View>
</View>
`;
